src/test/cypress/cypress/integration/Translation.spec.js

```diff
@@ -0,0 +1,338 @@
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ * This file is part of the OperatorFabric project.
+ */
+
+describe('Test translations', function () {
+    const ENGLISH = "en";
+    const FRENCH = "fr";
+    const DUTCH = "nl";
+
+    function changeLanguage(newLanguage) {
+        cy.get('#opfab-navbar-drop_user_menu').should("exist").click();
+        
+        // Wait for the menu to open
+        cy.get('.opfab-right-menu').should('exist');
+
+        cy.get('#opfab-navbar-right-menu-settings').should("exist").click();
+        cy.get('#opfab-setting-locale').select(newLanguage);
+        
+        // Wait for the language to be changed
+        cy.wait(500);
+    }
+
+    function setNightMode() {
+        cy.get('.opfab-right-menu').should('exist').then(menu => {
+            if (menu.find('#opfab-navbar-right-menu-night-mode').length > 0) {
+                cy.get('#opfab-navbar-right-menu-night-mode').click();
+            } else {
+                // Close the drop down user menu in order to always have it closed at the end of this function
+                cy.get('#opfab-navbar-drop_user_menu').should("exist").click();
+            }
+        })
+    }
+
+    function setDayMode() {
+        cy.get('.opfab-right-menu').should('exist').then(menu => {
+            if (menu.find('#opfab-navbar-right-menu-day-mode').length > 0) {
+                cy.get('#opfab-navbar-right-menu-day-mode').click();
+            } else {
+                // Close the drop down user menu in order to always have it closed at the end of this function
+                cy.get('#opfab-navbar-drop_user_menu').should("exist").click();
+            }
+        })
+
+    }
+
+    function checkMenuTitles(feedTitle, archivesTitle, monitoringTitle, loggingTitle, singleMenuTitle, secondMenuTitle, 
+                             firstEntryTitle, secondEntryTitle) {
+        cy.get("#opfab-navbar-menu-feed").should("have.text", feedTitle);
+        cy.get("#opfab-navbar-menu-archives").should("have.text", archivesTitle);
+        cy.get("#opfab-navbar-menu-monitoring").should("have.text", monitoringTitle);
+        cy.get("#opfab-navbar-menu-logging").should("have.text", loggingTitle);
+        cy.get("#opfab-navbar-menu-menu1").should("have.text", singleMenuTitle);
+        cy.get("#opfab-navbar-menu-dropdown-menu2").should("have.text", secondMenuTitle);
+        cy.get('#opfab-navbar-menu-dropdown-menu2').click();
+
+        // Test dropdown menus titles
+        cy.get('#opfab-navbar-menu-dropdown-menu2').click();
+        cy.get('.dropdown-menu').find('.text-link').eq(0).should("have.text", firstEntryTitle);
+        cy.get('.dropdown-menu').find('.text-link').eq(1).should("have.text", secondEntryTitle);
+    }
+
+    function checkRightMenuStaticEntries(realTimeTitle, settingsTitle, activityAreaTitle, notificationsTitle, aboutTitle,
```

Comment:
why not name notificationsTitle => feedConfigurationTitle ? to be coherent with #opfab-navbar-right-menu-feedconfiguration